docs/reference/docs/reindex.asciidoc

```diff
@@ -118,8 +118,11 @@ POST _reindex
 // CONSOLE
 // TEST[setup:twitter]
 
-By default, version conflicts abort the `_reindex` process, but you can just
-count them by setting `"conflicts": "proceed"` in the request body:
+By default, version conflicts abort the `_reindex` process. The `"conflicts"` request body
+parameter can be used to instruct `_reindex` to proceed with next document on version conflicts.
+Notice that the handling of other types of errors is unaffected by the `"conflicts"` parameter.
+By setting `"conflicts": "proceed"` in the request body the `_reindex` process will continue on version conflicts
```

Comment:
This is confusing because "By setting...the _reindex process will continue" seems to suggest that it is the process that is setting the parameter.

Suggestion:
When `"conflicts": "proceed"` is set in the request body, the `_reindex` process will continue on version conflicts

Reply:
Thanks, so good to have native speakers review docs, fixed in 7f76ba806de6513e68c3d183e3c01f05e3a484f6